curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/650046832f92c01a35834bca.md

```diff
@@ -0,0 +1,407 @@
+---
+id: 650046832f92c01a35834bca
+title: Step 56
+challengeType: 0
+dashedName: step-56
+---
+
+# --description--
+
+You also want a deleted task to be deleted in local storage. For this, you don't need the `removeItem()` or `clear()` methods. Since you already use `splice()` to remove the deleted task from `taskData`, all you need now is to save `taskData` in local storage again.
```

Comment:
```suggestion
You also want a deleted task to be removed from local storage. For this, you don't need the `removeItem()` or `clear()` methods. Since you already use `splice()` to remove the deleted task from `taskData`, all you need to do now is save `taskData` to local storage again.
```